src/java.desktop/share/native/common/awt/debug/debug_trace.h

```diff
@@ -62,6 +62,9 @@ JNIEXPORT void JNICALL DTrace_VPrint(const char * file, int line, int argc, cons
 JNIEXPORT void JNICALL DTrace_VPrintln(const char * file, int line, int argc, const char * fmt, va_list arglist);
 
 /* each file includes this flag indicating module trace status */
+#ifdef __cplusplus
+[[maybe_unused]]
```

Comment:
I don't really know what to do about this one. The solution is messy, but it's the only one I could come up with. A static global in a header is just begging for an unused\-variable error to happen

Reply:
Hi Thomas, I don't think that is possible due to how it's used, from what I can tell from reading the code this global is meant to be unique to each file that the header is included in, as the comment says:

> /* each file includes this flag indicating module trace status */

I've been trying to think of alternatives to the current "Global unique to each file defined inside a header" approach currently taken here, but alas I couldn't really think of anything